csvtotable/convert.py

```diff
@@ -130,9 +130,10 @@ def render_template(table_headers, table_items, **options):
     virtual_scroll = False
     scroll_y = height
 
-    if virtual_scroll_limit != -1 and len(table_items) > virtual_scroll_limit:
-        virtual_scroll = True
-        display_length = -1
+    if virtual_scroll_limit:
```

Comment:
```
import pyexcel as p
sheet = p.get_sheet(file_name='goog.ods')
sheet.save_as('me.sortable.html', display_length=10)
```

Above code would fail  and get this:

> /Users/jaska/github/data-analysis-env/lib/python3.5/site-packages/csvtotable/convert.py in render_template(table_headers, table_items, **options)
>     131     scroll_y = height
>     132 
> --> 133     if virtual_scroll_limit != -1 and len(table_items) > virtual_scroll_limit:
>     134         virtual_scroll = True
>     135         display_length = -1
> 
> TypeError: unorderable types: int() > NoneType()